js/utils/resyncBlockchain.js

```diff
@@ -170,14 +172,21 @@ export default function resyncBlockchain(coinType) {
           lastBlockchainResync = typeof lastBlockchainResync === 'object' ?
             lastBlockchainResync : {};
           lastBlockchainResync[coinType] = (new Date()).getTime();
-          _server.save({ lastBlockchainResync })
-            .done(() => {
+          const serverSave = _server.save({ lastBlockchainResync });
```

Comment:
So the problem was that `isjs` was updated and it looks like it not longer considers `localhost` as a valid IP, so I had to manually add that condition to the mode validation.

What happens is `Model.save()` returns `false` if validation fails as opposed to the jqXhr promise if it passes validation. So handling the case here. It really shouldn't ever fail validation unless it's something weird like this and if it does, it will have better console output than the seemingly random exception that was in place before.